.gitignore

```diff
@@ -99,5 +99,5 @@ typings/
 # TernJS port file
 .tern-port
 
-# seeding files
-pollinator.js
+# bundle
+client/dist/bundle.js
```

Comment:
Linter wants new line here and in a couple other places. Check other files.

Reply:
Will do.